src/app/Application.java

```diff
@@ -0,0 +1,85 @@
+package app;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.InputReader;
+import model.ExpenseMode;
+import model.User;
+import model.UserRegistry;
+import utils.DisplayUtils;
+import utils.ExpenseUtils;
+import utils.FormatUtils;
+
+public class Application {
+
+  public static void main(String[] args) {
+    User u1 = new User("u1", "u1@gmail.com", "81924305812");
+    User u2 = new User("u2", "u2@gmail.com", "83624265812");
+    User u3 = new User("u3", "u3@gmail.com", "91465305812");
+    User u4 = new User("u4", "u4@gmail.com", "97924345852");
+    UserRegistry.addUser(u1);
```

Comment:
Don't hardcode the users . Either import it from a file or expose an function/interface to add users .

Reply:
It is actually fine to do so in the machine coding interview based on my experience.

Reply:
Thanks for the review and thanks a lot for conducting this @gcnit , it really helped a lot. Based on this design, given that the functionality was complete, do you think this solution would be accepted in a machine coding round ?

Reply:
Design is good but could have been simplified.